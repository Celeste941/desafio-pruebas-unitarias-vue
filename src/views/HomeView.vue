<template>
    <section>
      <h1>Secci√≥n de Home</h1>
    </section>
  </template>

<script>
export default {
  name: 'HomeView'
}
</script>