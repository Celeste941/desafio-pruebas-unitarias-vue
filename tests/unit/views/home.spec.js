import { mount } from "@vue/test-utils";
import HomeView from "@/views/HomeView.vue";

describe("Test de vista Home", () => {
  it("Match de snapshot con HomeView.vue", () => {
    const wrapper = mount(HomeView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});