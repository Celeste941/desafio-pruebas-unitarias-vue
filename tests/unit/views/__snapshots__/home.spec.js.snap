// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de vista Home Match de snapshot con HomeView.vue 1`] = `
<section>
  <h1>Secci√≥n de Home</h1>
</section>
`;
